File: returned/python/today16/course_schedule_test.py
from course_schedule import Solution

def test1():
    assert Solution().canFinish(2, [[1,0]]) == True

def test2():
    assert Solution().canFinish(2, [[1,0], [0,1]]) == False

def test3():
    assert Solution().canFinish(3, [[0,2],[1,2],[2,0]]) == False

File: csharp/LeetTests/Today4/CourseScheduleTest.cs
using Today4.CourseSchedule;
namespace Today4;

internal class CourseScheduleTest
{
    [Test]
    public void Test1()
    {
        var solution = new Solution();
        var result = solution.CanFinish(2, [[1, 0]]);
        Assert.That(result, Is.EqualTo(true));
    }

    [Test]
    public void Test2()
    {
        var solution = new Solution();
        var result = solution.CanFinish(2, [[1, 0], [0, 1]]);
        Assert.That(result, Is.EqualTo(false));
    }
}